app-backend/api/build.sbt

```diff
@@ -1,51 +1,3 @@
 name := "api"
 
 assemblyJarName in assembly := "api-assembly.jar"
-
-initialCommands in console := """
```

Comment:
I agree about the unused import warning being a problem - but that's still a problem after this PR right? If I import anything I still get the error.

Reply:
You still get a warning, but the current problem is that you can't even initialize the `console` session, while after this PR, you can start the console, and _then_ it warns you about some things.